eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -192,7 +192,7 @@ stages:
                   Cloud: ${{ cloud.key }}
 
   # The Prerelease and Release stages are conditioned on whether we are building a pull request and the branch.
-  - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), not(contains(variables['Build.DefinitionName'], 'weekly')), eq(variables['System.TeamProject'], 'internal'))}}:
+  - ${{if and(in(variables['Build.Reason'], 'Manual', ''), not(contains(variables['Build.DefinitionName'], 'weekly')), eq(variables['System.TeamProject'], 'internal'))}}:
```

Comment:
What is empty Build.Reason?

Reply:
According to past you (https://github.com/Azure/azure-sdk-for-net/pull/24132): 

> When you are trying to queue a manual run DevOps sets Build.Reason to empty which blocks the full generation of stages in the stage selection UI. To work around this we check for Build.Reason empty or Manual. This allows for teams to uncheck a release stage for any packages that they don't plan to release.

This is similar to how it is defined in the other repos (e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/stages/archetype-net-release.yml#L39).